repos/common/topics/systeminfo.py

```diff
@@ -0,0 +1,5 @@
+from leapp.topics import Topic
+
+
+class SystemInfoTopic(Topic):
```

Comment:
We should make this available here instead. This topic already exists in a different repo, we would just have to move it, otherwise it would break other things.

Reply:
Topic deleted from the other repo.